examples/sqlite/src/main.rs

```diff
@@ -223,11 +228,13 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .with_secure(false)
         .with_expiry(Expiry::OnInactivity(Duration::days(1)));
 
+    let db_path = std::path::Path::new(&std::env::var("CARGO_MANIFEST_DIR")?).join("example.db"); // TODO: This isn't exemplary.
+
     // Auth service.
     //
     // This combines the session layer with our backend to establish the auth
     // service which will provide the auth session as a request extension.
-    let pool = SqlitePool::connect("sqlite:examples/example.db").await?;
+    let pool = SqlitePool::connect(&db_path.to_string_lossy()).await?;
```

Comment:
Since this is a separate project now I'd just add sqlx migrations and make this database in-memory. We'd get rid of that example.db file, which is binary and sticks out weirdly.

Reply:
Great idea. Thank you!